catalog/core/catalog-core-validationparser/src/test/groovy/ddf/catalog/validation/impl/ValidationParserSpec.groovy

```diff
@@ -58,8 +51,8 @@ class ValidationParserSpec extends Specification {
 
         defaultAttributeValueRegistry = new DefaultAttributeValueRegistryImpl()
 
-        validationParser = new ValidationParser(attributeRegistry, attributeValidatorRegistry,
-                defaultAttributeValueRegistry)
+        validationParser = spy(new ValidationParser(attributeRegistry, attributeValidatorRegistry,
```

Comment:
✏️ Should be able to use Spock's native `MockingApi.Spy` factory method here instead of using Mockito. It's fine as it is, but would be more idiomatic that way.

Reply:
Fixed